### PR TITLE
refactor(forms): change control suffix to fieldff

### PR DESCRIPTION
**Pull request by ismay**
Todo:

- [x] Rename *Control components in the forms workspace to *FieldFF (e. g. InputField in widgets; InputFieldFF in forms)
- [x] Rename FieldSetField to FieldGroup in widgets and GroupControl to FieldGroupFF

### CODE REVIEW

File: cypress/integration/forms/SingleSelect/common/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Given } from 'cypress-cucumber-preprocessor/steps'
```
Comment (Mohammer5) [inline cypress/integration/forms/SingleSelect/common/index.js:1]: The file path doesn't match the component's name

File: cypress/integration/forms/Switch/can_toggle_a_boolean/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import '../common/index.js'
 import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
```
Comment (Mohammer5) [inline cypress/integration/forms/Switch/can_toggle_a_boolean/index.js:2]: The file path doesn't match the component's name

File: cypress/integration/forms/Switch/can_toggle_a_value/index.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import '../common/index.js'
 import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
```
Comment (Mohammer5) [inline cypress/integration/forms/Switch/can_toggle_a_value/index.js:2]: The file path doesn't match the component's name

File: cypress/integration/forms/Switch/displays_error/index.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
 import { hasValueMessage } from '../../../../../packages/forms/src/validators/hasValue.js'
```
Comment (Mohammer5) [inline cypress/integration/forms/Switch/displays_error/index.js:3]: The file path doesn't match the component's name

**Comment by ismay** [comment]
@Mohammer5 I've updated the paths (54261de), that's what you meant right?

**Pull request merged**